examples/SB3_quantstats.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
Please also add SB3_quantstats.html and remove quantstats-tearsheet.html

Reply:
`qs.reports.html(returns, output='SB3_quantstats.html')`

but the file 'quantstats-tearsheet.html' is created again, why?
I copied your a2c example 4 months ago and adjusted only a few lines. 

I am not familiar with 'quantstats' at all. The graphics look identical. Better also remove 'SB3_quantstats.ipynb'?

Reply:
I think 'SB3_quantstats.ipynb' is a good example, which was requested by one the users when I started this project, and we should keep it. About the 'quantstats-tearsheet.html' problem, I can work on it later if you haven't fixed it yet.

Reply:
I have updated your a2c example, no longer compatible with SB3.

'quantstats' has a parameter for filename. Probably new version necessary.
https://github.com/ranaroussi/quantstats/blob/main/quantstats/reports.py#L63